src/actions/songs/recommend-songs.js

```diff
@@ -0,0 +1,87 @@
+const song = require("../../models").song,
+	genre = require("../../models").genre,
+	user = require("../../models").user,
+	listened = require("../../models").previously_listened,
+	songResult = require("../../models").likert_template_result,
+	maxSongs = 10;
+
+var songs, temp = [], amountResults
+
+module.exports = (genreId, moodId) => {
+	listenedMusic(genreId, moodId)
+	newMusic(genreId, moodId)
+	return songs;
+	// return genre.findOne({
+	// 	where: {
+	// 		GenreId: genreId
+	// 	}
+	// }).then(result => {
+	// 	if(result === null){
+	// 		return {
+	// 			ok: false,
+	// 			message: 'Genre could not be found.'
+	// 		};
+	// 	}
+	// 	return song.findAll({
+	// 		where: {
+	// 			Mood_MoodId: result.genreId
+	// 		}
+	// 	}).then(result => {
+	// 		return {
+	// 			ok: true,
+	// 			message: result
+	// 		}
+	// 	});
+	// });
+};
+
+function newMusic(genreId, moodId) { //, artistId, albumId){
+	song.findall({
+		where: {
+			Mood_MoodId: moodId,
```

Comment:
Mood_Moodid wordt niet meer gebruikt in het tabel, als dit wel nog zo is dient dit veranderd te worden naar moodId. Tevens genre.